astropy_helpers/sphinx/ext/automodapi.py

```diff
@@ -52,6 +52,14 @@
         given, only objects that are actually in a subpackage of the package
         currently being documented are included.
 
+    * ``:inherited-members: True | False``
```

Comment:
Oh, and it should probably be stated explicitly that you should use ``:no-inherited-members:`` to negate a default set to True.  Maybe just have the bullet point be ```* ``:inherited-members:`` / ``:no-inherited-members:`` ```